pkg/manifests/types.go

```diff
@@ -19,377 +19,398 @@ import (
+// instance used for user-defined projects.
 type AlertmanagerUserWorkloadConfig struct {
-	// Enabled a boolean flag to enable or disable a dedicated instance of Alertmanager
-	// for user-defined projects under openshift-user-workload-monitoring
-	// default: false
+	// A Boolean flag that enables or disables a dedicated instance of 
+	// Alertmanager for user-defined projects in the `openshift-user-workload-monitoring` namespace.
+	// The default value is `false`.
 	Enabled bool `json:"enabled,omitempty"`
-	// EnableAlertmanagerConfig a boolean flag to enable or disable user-defined namespaces to be selected
-	// for AlertmanagerConfig lookup, by default Alertmanager only looks for configuration
-	// in the namespace where it was deployed to
-	// default: false
+	// A Boolean flag to enable or disable user-defined namespaces to be selected
+	// for `AlertmanagerConfig` lookup. By default, Alertmanager only looks for configuration
+	// in the namespace to which it was deployed.
+	// The default value is `false`.
 	EnableAlertmanagerConfig bool `json:"enableAlertmanagerConfig,omitempty"`
-	// LogLevel defines the log level for Alertmanager.
-	// Possible values are: error, warn, info, debug.
-	// default: info
+	// Defines the log level setting for Alertmanager for user workload monitoring.
+	// The possible values are `error`, `warn`, `info`, and `debug`.
+	// The default value is `info`.
 	LogLevel string `json:"logLevel,omitempty"`
-	// Resources define resources requests and limits for single Pods.
+	// Defines resource requests and limits for single pods.
 	Resources *v1.ResourceRequirements `json:"resources,omitempty"`
-	// NodeSelector defines which Nodes the Pods are scheduled on.
+	// Defines the nodes on which the pods are scheduled.
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
-	// Tolerations defines the Pods tolerations.
+	// Defines tolerations for the pods.
 	Tolerations []v1.Toleration `json:"tolerations,omitempty"`
-	// VolumeClaimTemplate defines persistent storage for Alertmanager. It's possible to
-	// configure storageClass and size of volume.
+	// Defines persistent storage for Alertmanager. Use this setting to
+	// configure the storage class and size of a volume.
 	VolumeClaimTemplate *monv1.EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate,omitempty"`
 }
 
-// PrometheusRestrictedConfig defines configuration related to the Prometheus component that
-// will monitor user-defined projects.
+// `PrometheusRestrictedConfig` defines the settings for the Prometheus 
+// component that monitors user-defined projects.
 type PrometheusRestrictedConfig struct {
-	// AlertmanagerConfigs holds configuration about how the Prometheus component should communicate
-	// with aditional Alertmanager instances.
-	// default: nil
+	// Configures how the Prometheus component communicates
+	// with additional Alertmanager instances.
+	// The default value is `nil`.
```

Comment:
Not sure if mentioning the default behavior is useful. It's kind of implicit IMHO.

```suggestion
	// Configures additional Alertmanager instances to which the Prometheus component sends user-defined alerts.
	// No instance is configured by default.
```

Reply:
fixed